src/components/reaction/reaction.js

```diff
@@ -1,34 +1,94 @@
-import React, { useState } from 'react';
+/* eslint-disable no-console */
+import React, { useState, useEffect } from 'react';
 import { faThumbsUp, faThumbsDown } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { useAuth0 } from '@auth0/auth0-react';
+import PropTypes from 'prop-types';
+import {
+  GetLikeDislikeForUser,
+  PostReactionForUser,
+  ReactionType,
+} from '../../services/apiService';
+
+const Reaction = (props) => {
+  const ruleId = props.ruleId; // <- extracts fields from props
```

Comment:
typical js syntax would have us destructuring the object here instead of using regular dot notation.
e.g.
`const { ruleId } = props;`
This lets us pull multiple params from props easily and is a good habit to get into